Added documentation for optional color configurations on Android. (#1566)
beeware/briefcase: docs/reference/platforms/android/gradle.rst

Android projects do not support installer images.

Colors
======

> In addition to ``splash_background_color``, Android projects allow for
> configuration of an ``accent_color``, ``primary_color``, and
> ``primary_color_dark``.

Reviewer: There's no need to list all the things that can be configured just before you list all the things that can be configured :-)
```suggestion
Android allows for some customization of the colors used by your app:
```

> * ``accent_color`` is used subtly throughout an Android app to call attention
>   to key elements. It's used on things like form labels and inputs.

Reviewer: ```suggestion
* ``accent_color`` is used as a subtle highlight throughout your app to call attention
  to key elements. It's used on things like form labels and inputs.
```

Author: I made all those changes. Thanks for helping me get this in shape!
